README.md

```diff
@@ -1,70 +1,52 @@
-# Azure SDK for Javascript
+# Azure SDK for JavaScript
+
 
 | Component            | Build Status                                                                                                                                                                                |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Management Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/138?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=138&branchName=master) |
 | Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/45?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=45&branchName=master)   |
 
-This project provides an isomorphic Javascript package with TypeScript definitions that makes it easy to consume and manage
-Microsoft Azure Services.
-It supports SDKs for:
-
-- ARM services (control plane) (packages with the naming convention of `@azure/arm-<servicename>`)
-- data plane of some Azure services (packages with the naming convention of `@azure/<servicename>`).
-
-## Documentation
-
-Documentation of the supported SDKs can be found here:
-
-- https://docs.microsoft.com/en-us/javascript/azure - This website primarily provides SDK documentation for
-
-  - ARM based services (`@azure/arm-<serviceName>`)
-  - data plane SDKs like `@azure/batch`, `@azure/graph`, etc.
+This repository contains official JavaScript libraries for Azure services. For documentation go to [Azure SDK for JavaScript documentation](https://aka.ms/js-docs).
```

Comment:
We have had feedback from existing customers regarding the confusion between the libraries from azure-sdk-for-node repo and this one.

We can benefit from having a line here on how this repo is different from the old one and link to the migration guide.